docs/ci.md

```diff
@@ -8,7 +8,7 @@ To generate this Matrix run `./tests/scripts/md-table/main.py`
 |---| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
 amazon |  :white_check_mark: | :x: | :x: | :x: | :x: | :x: | :x: | :x: | :x: | :x: |
 centos7 |  :white_check_mark: | :x: | :x: | :x: | :x: | :x: | :white_check_mark: | :x: | :x: | :white_check_mark: |
```

Comment:
Doesn't this work cover CentOS7?

Reply:
It doesn't, CI matrix need to only have the specified OS

Reply:
Thanks for your explanation @floryut 
I got it.
I cannot find any CentOS8 specific on this PR, hopefully that could work for CentOS7 also.

Reply:
We tested with centos8 and ubuntu 18.04 distro. We enabled the CI test for centos8 and ubuntu18.04 distro. It should work with centos7 as well. But having CI job for Centos8 and ubuntu 18.04 distro is the minimum CI requirement.